### DISCUSSION TITLE
Option to clear other cached data like MDL

### DISCUSSION THREAD

**Discussion by lilggamegenius** [CapsAdmin/pac3#623] [Ideas]
like #603 but for other parts

**Comment by CapsAdmin** [comment]
This is actually not possible because once a gma archive has been mounted you can't delete it. You can work around it by salting the hash that pac3 generates for it though.

**Comment by lilggamegenius** [comment]
Maybe you could add some option that means that you're currently editing the model so it stores it in a special temp folder that increments a counter. This way you can always load a new version of the model.
Similar to `pac3_cache/testing/somehash-test1/<name>.mdl` then you could clear the testing folder when pac starts or something.
